docs/src/main/sphinx/connector/query-passthrough-warning.fragment

```diff
@@ -0,0 +1,11 @@
+
+.. Warning::
+
+    The native query passed to the underlying data source is required to return
+    a table as a result set. This can be a one column table with one row. No
```

Comment:
Lets move the sentence to the end of the paragraph and modify to  "No validation or security checks of these queries are performed by Trino. Only the data source performs these task using its own configuration."

Reply:
I will take on these notes, but should I also format the section as it was previously, with a rest of the section just as paragraph and these last two sentences as the warning? It seems to me that the rest is just explaining the workaround process and should not be included in the warning.